requirements.txt

```diff
@@ -1,33 +1,72 @@
-ddtrace
-diff-match-patch==20181111
-django-bootstrap3==11.0.0
-django-debug-toolbar==1.11.1
-django-enumfield==1.5
-django-environ==0.4.5
-django-filter==2.4.0
+asgiref==3.6.0
```

Comment:
This really feels like a `pip freeze > requirements.txt`. I think we would be better off using [`pip-tools`](https://pip-tools.readthedocs.io/en/latest/)'s pip-compile. That way we can keep the `requirements.in` clean with only packages we require ourselves and let `pip-compile` handle the dependencies of those packages (and fix their version)

Reply:
fixed